modules/microshift-restoring-data-backups.adoc

```diff
@@ -0,0 +1,30 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restoring-data-backups-manually_{context}"]
+= Restoring application data backups manually
+
+If you are not using an OSTree system, you can restore application data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When restoring a backup, you must use the entire file path.
+
+.Prerequisites
+* Root access to the host.
+* Full path of the data backup file.
+* {product-title} is stopped.
+
+[IMPORTANT]
+====
+{product-title} must be stopped during the backup process to avoid data corruption.
+====
+
+.Procedure
+. Manually restore an application backup file using the entire file path of the backup you want to restore by using the following command syntax:
++
+[source, terminal]
+----
+$ sudo microshift restore </var/lib/microshift-backups/my-4.14-backup>
```

Comment:
In other places it's `# microshift`, not `$ sudo microshift`

Reply:
Yeah, I think in another PR we talked about making everything just `$ sudo` because we have mixed use in the Docs. If OK here, I'll just use `$ sudo`.